src/api/v1/db/oracledb/setupDb.sql

```diff
@@ -0,0 +1,72 @@
+DROP TABLE PIZZA_INGREDIENTS;
+DROP TABLE PIZZAS;
+DROP TABLE INGREDIENTS;
+DROP TABLE DOUGHS;
+
+CREATE TABLE "DOUGHS" 
+   (	"ID" NUMBER GENERATED ALWAYS AS IDENTITY MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER  NOCYCLE  NOT NULL, 
+	"NAME" VARCHAR2(256 BYTE) NOT NULL, 
```

Comment:
I've used 256 bytes for "short" strings, like names, and 512 bytes for "long" strings like descriptions.